pandas/tests/scalar/test_nat.py

```diff
@@ -104,8 +104,13 @@ def test_identity(klass, value):
 @pytest.mark.parametrize("klass", [Timestamp, Timedelta, Period])
 @pytest.mark.parametrize("value", ["", "nat", "NAT", None, np.nan])
 def test_equality(klass, value):
```

Comment:
I would instead remove the empty and put this in a new test for that case (and assert the error)

these are duplicating the constructor tests so should have either / or (i think they are fine here)